### DISCUSSION TITLE
在使用vue3版本的antvs2尝试获取单元格位置时，位置获取不带行头和列头偏移量的问题

### DISCUSSION THREAD

**Discussion by Xugen-Ma** [antvis/S2#2229] [Q&A] [answered]
我在vue中sheetcomponent标签中指定了点击单元格事件，并且在事件中尝试获取单元格的x,y坐标，发现并不如预期获取的单元格相对于canvas左上角的坐标，而是获取的单元格相对角头右下角的坐标。我不清楚这个是一个bug，还是需要在一开始就获取元素的列头的高度和行头的宽度，并将其作为偏移量加入到获取的坐标中。希望能够得到解答。
`<SheetComponent :dataCfg="dataCfg" :options="options" @data-cell-click="handleCellClick" />`
`const meta = targetCellInfo.target.getMeta()`
[最小可复现样例](https://codesandbox.io/s/dreamy-panini-qlvt0u?file=/src/App.vue:4675-4719)
版本信息：
vue: 3.2.33 antvs2:1.47.1 antvs2-vue: 1.0.1 ant-design-vue:3.2.16

**Comment by lijinke666** [comment, marked as answer]
dateCell 内部的坐标基准点是基于箭头所指位置, `getMeta()` 取到的 `x, y` 是相对位置

![image](https://github.com/antvis/S2/assets/21015895/d1da7387-dcc6-440b-87bf-a04f0f98cff8)

如果你是想获取鼠标点击的坐标, 即原始 Event 的坐标, 你可以取 `clientX` 和 `clientY`

![image](https://github.com/antvis/S2/assets/21015895/50e861e1-730f-4198-8821-6b467f1e1237)
